clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/DockerRegistryConfiguration.groovy

```diff
@@ -48,4 +51,9 @@ class DockerRegistryConfiguration {
   DockerRegistryHealthIndicator dockerRegistryHealthIndicator() {
     new DockerRegistryHealthIndicator()
   }
+
+  @Bean
+  String dockerApplicationName(@Value('${Implementation-Version:Unknown}') String implementationVersion) {
```

Comment:
Might be just about time to 'promote' this factory method out of GCE and Kubernetes and into core.